Modules/CTP/include/CTP/TH1ctpReductor.h

```diff
@@ -18,7 +18,7 @@
 
 #include "QualityControl/Reductor.h"
 
-namespace o2::quality_control_modules::ctp
+namespace o2::quality_control::postprocessing
```

Comment:
Since this Reductor is CTP-specific, I would keep it within the `o2::quality_control_modules::ctp` namespace.